script/release/DSL-scripts/KIE_Release.sh

```diff
@@ -0,0 +1,622 @@
+def pushReleaseBranches =
```

Comment:
We need to re-think how to create these jobs. This is definitely much better than updating the jobs via JenkinsUI. It clears a lot of things, but it still contains a lot of duplication which would be great to get rid of.